### PR TITLE
Ability to mention user in forum

### PR DESCRIPTION
**Pull request by Regaron**
Addresses: #1173 

Demo:

https://user-images.githubusercontent.com/18470532/199933805-7669b727-cd76-43ad-9125-c8be379973a6.mov

https://user-images.githubusercontent.com/18470532/200725154-1e9ce5f7-c7ed-4ec3-8b46-9c9bc232567f.mov

### CODE REVIEW

File: app/models/forum_thread.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ class ForumThread < ApplicationRecord
   has_many :optin_subscribers, -> { where(forum_subscriptions: {subscription_type: :optin}) }, through: :forum_subscriptions, source: :user
   has_many :optout_subscribers, -> { where(forum_subscriptions: {subscription_type: :optout}) }, through: :forum_subscriptions, source: :user
   has_many :users, through: :forum_posts
-
+
```
Comment (alis-khadka) [inline app/models/forum_thread.rb:12]: nit: Please remove such trailing spaces.

**Comment by donrestarone** [comment]
Code looks good! @Regaron can you add a test case to verify that `@mentioned` users get notified?

**Comment by donrestarone** [comment]
@Regaron any updates on this?

**Comment by donrestarone** [comment]
@kripananda-yadav this is ready for UAT. Please file under Nikean.org and note we are looking to ship this feature before the end of the month

**Comment by kripananda-yadav** [comment]
@donrestarone  @Regaron  I don't see the Review app for the testing purpose Or is there any other way to test it? Please let me know.

**Pull request merged**